### PR TITLE
User can view order's tracking status

### PR DESCRIPTION
**Pull request by hieuphq**
#### Status

READY

#### What's this PR do?

- [x] Search order with order code
- [x] User can input order code in main page
- [x] User can view order detail

#### Where should the reviewer start?

web\controllers\tracking_controller.ex
web\models\order.ex
web\models\partner.ex
web\models\voucher.ex
web\templates\tracking\*
web\views\tracking_view.ex

#### Screenshots (if appropriate)

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -12,8 +12,8 @@ config :logger, level: :warn
 # Configure your database
 config :delivr, Delivr.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
+  username: "hieuphan",
+  password: "Congamai1",
```
Comment (tieubao) [inline config/test.exs]: sao Hiếu lại quyết định set thông tin cá nhân vào đây?

File: web/controllers.ex [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+				format: "%n %u", precision: 0
+			)
+		else
+			number_to_currency(
+	      0, unit: "đ",
+	      separator: ",", delimiter: ".",
+	      format: "%n %u", precision: 0
+	    )
+		end
+
```
Comment (hieuphq) [inline web/controllers.ex]: Xấu quá

**Comment by thminhVN** [comment]
👍

**Pull request merged**